github-sample/workflows/build-publish.yml

```diff
@@ -301,6 +270,42 @@ jobs:
       run: |
         echo $docker_digest
         echo "digest=$docker_digest" >> $GITHUB_OUTPUT
+  docker-build-auth-predeployment:
+    name: Build Auth Predeployment image
+    timeout-minutes: 20
+    runs-on: ubuntu-latest
+    needs:
+    - version
+    steps:
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v2
+    - name: Login to GitHub Container Registry
+      uses: docker/login-action@v2
+      with:
+        registry: ghcr.io
+        username: ${{ github.repository_owner }}
+        password: ${{ secrets.GITHUB_TOKEN }}
+    - name: Docker meta
+      id: meta
+      uses: docker/metadata-action@v3
+      with:
+        images: name/app
+        labels: org.opencontainers.image.version=candidate-${{ needs.version.outputs.app_version }}
+    - name: Build Auth Predeployment image
+      uses: docker/build-push-action@v3
+      id: docker-build-auth-predeployment
+      with:
+        file: Dockerfile
+        push: true
+        load: false
+        labels: ${{ steps.meta.outputs.labels }}
+        tags: ghcr.io/covergo/auth-predeployment:candidate-${{ needs.version.outputs.app_version }}
+        cache-from: type=registry,ref=ghcr.io/covergo/auth-predeployment-cache:${{ needs.version.outputs.issue_id_slug }}
+        cache-to: type=registry,ref=ghcr.io/covergo/auth-predeployment-cache:${{ needs.version.outputs.issue_id_slug }},mode=max
+        target: build-predeployment
+        build-args: COMMIT_SHA=${{ github.sha }}
+    - name: Image digest
+      run: echo ${{ steps.docker-build-auth-predeployment.outputs.digest }}
```

Comment:
This should match with line 272. Otherwise line 995 will not get the digest value.